site/gatsby-site/src/pages/apps/incidents.js

```diff
@@ -18,12 +18,7 @@ export default function IncidentsPage(props) {
         <title>{t('Incidents')}</title>
       </AiidHelmet>
       <div className="bootstrap">
-        {!incidentsData && (
-          <div className="p-4 flex justify-center align-items-center gap-2">
-            <Spinner />
-            <Trans>Fetching Incidents...</Trans>
-          </div>
-        )}
+        {!incidentsData && <ListSkeleton />}
```

Comment:
The left edge seems to get cut off here:

![image](https://user-images.githubusercontent.com/25443411/205671087-61b10f96-459f-4b6a-a321-da6f2d8db022.png)